src/laser_utils.cpp

```diff
@@ -99,7 +99,28 @@ karto::LaserRangeFinder * LaserAssistant::makeLaser(const double & mountingYaw)
     karto::LaserRangeFinder_Custom, karto::Name("Custom Described Lidar"));
   laser->SetOffsetPose(karto::Pose2(laser_pose_.transform.translation.x,
     laser_pose_.transform.translation.y, mountingYaw));
-  laser->SetMinimumRange(scan_.range_min);
+
+  double min_laser_range = 25;
```

Comment:
`0.0`

Reply:
Thanks, have updated it in the latest commit